.github/workflows/ci-dgraph-code-coverage.yml

```diff
@@ -14,7 +14,7 @@ on:
       - main
       - 'release/**'
   schedule:
-    - cron: "0 */2 * * *" # every 2hrs
+    - cron: "0 0,6,12,18 * * *"
```

Comment:
whats the difference in the syntax ? can you just say `0 */2` if you are changing it to every 6 hours.